src/server/list_family.cc

```diff
@@ -1315,14 +1315,13 @@ void ListFamily::Register(CommandRegistry* registry) {
       << CI{"LLEN", CO::READONLY | CO::FAST, 2, 1, 1, 1}.HFUNC(LLen)
       << CI{"LPOS", CO::READONLY | CO::FAST, -3, 1, 1, 1}.HFUNC(LPos)
       << CI{"LINDEX", CO::READONLY, 3, 1, 1, 1}.HFUNC(LIndex)
-      << CI{"LINSERT", CO::WRITE, 5, 1, 1, 1}.HFUNC(LInsert)
+      << CI{"LINSERT", CO::WRITE | CO::DENYOOM, 5, 1, 1, 1}.HFUNC(LInsert)
```

Comment:
This can increase memory usage, right? Why allow it then?

Reply:
Reply to all comments above: Its flipped 😄 Deny -> deny